src/main/java/uk/ac/ic/wlgitbridge/server/Oauth2Filter.java

```diff
@@ -200,4 +203,23 @@ private void unauthorized(
         w.close();
     }
 
+    private void missing(
```

Comment:
Nitpick: maybe clearer to call this something like `respondWithNotFound` or `handleMissingRepository`?